iam-roles-policies.tf

```diff
@@ -104,9 +104,12 @@ data "aws_iam_policy_document" "codepipeline_policy_terraform" {
       "codebuild:StartBuild"
     ]
 
-    resources = ["*"]
+    resources = [
+      var.cb_accounts_map[*].account_id,
+    ]
   }
 
+  #trivy:ignore:AVD-AWS-0057
```

Comment:
```suggestion

```
What do you think about removing this ignore and add explicitly the permissions needed? It's not a long list.

https://github.com/StratusGrid/terraform-aws-multiaccount-pipeline/pull/20/files#diff-43bdd3b1698eec8a0860e5953b4534525a906fc7c683590324a5774c6611d66dR119
```
"ReEncryptFrom",
"ReEncryptTo",
```

https://github.com/StratusGrid/terraform-aws-multiaccount-pipeline/pull/20/files#diff-43bdd3b1698eec8a0860e5953b4534525a906fc7c683590324a5774c6611d66dR120
```
"GenerateDataKey",
"GenerateDataKeyPair",
"GenerateDataKeyPairWithoutPlaintext",
"GenerateDataKeyWithoutPlaintext",

```



reference:
https://docs.aws.amazon.com/service-authorization/latest/reference/list_awskeymanagementservice.html

Reply:
the * substitutes the permission set, https://docs.aws.amazon.com/kms/latest/APIReference/API_ReEncrypt.html